### PR TITLE
Typescript definition file added

### PR DESCRIPTION
**Pull request by vanhumbeecka**
Related Github issue: #842

### CODE REVIEW

**Comment by amaurymartiny** [comment]
I believe you should add a `types` field in package.json pointing to index.d.ts: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "eslint": "^4.2.0",
     "eslint-config-mourner": "^2.0.1",
     "express": "^4.13.4",
+    "geojson": "^0.5.0",
```
Comment (amaurymartiny) [inline package.json]: This should be added via `yarn add geojson`, right now the yarn.lock is not updated

Comment (vanhumbeecka) [reply package.json]: Got it

**Comment by ansis** [comment]
Hey! Thanks for your interest and contributing these. Since we can't commit to maintaining these definitions I think it would be best for us not to merge it. Previously we have had users using mapbox-gl commit typescript definitions for that to DefinitelyTyped (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11386). Would that approach work for you?

I'd be happy to merge a change to the readme that mentions that typescript definitions are available through DefinitelyTyped.

**Comment by kkaefer** [comment]
Yep, DefinitelyTyped is the where these should go. Thanks for your contribution!

**Comment by JamesLMilner** [comment]
@vanhumbeecka did you ever get a chance to raise this on DefinitelyTyped? I couldn't see anything in the history: https://github.com/DefinitelyTyped/DefinitelyTyped/pulls?utf8=%E2%9C%93&q=is%3Apr+mapboxgl

**Comment by vanhumbeecka** [comment]
@JamesLMilner no i couldn't. So far I still just use these hardcoded in a separate `types` folder and point my `tsconfig.json` file to it for extra type source. Works well enough.

**Comment by JamesLMilner** [comment]
@vanhumbeecka okay, thanks for getting back to me. Any chance you could share your up to date types here or in a Gist perhaps? I could look into raising it longer term

**Comment by chriszrc** [comment]
I'm not seeing this in DefinitelyTyped:

https://github.com/DefinitelyTyped/DefinitelyTyped/search?q=mapboxdraw

Is this up yet? Is there some reason it can't be added?

**Pull request closed without merging**